InvenTree/templates/base.html

```diff
@@ -121,6 +121,7 @@
     </div>
     {% include 'modals.html' %}
     {% include 'about.html' %}
+    {% include "notifications.html" %}
```

Comment:
Looks like pollution from your other PR?

Reply:
I hate it when that happens to me, thanks for the heads up!